app/services/widget-settings/widget-settings.ts

```diff
@@ -107,8 +107,12 @@ export abstract class WidgetSettingsService<TWidgetData extends IWidgetData> ext
   protected handleDataAfterFetch(data: any): TWidgetData {
 
     // patch fetched data to have the same data format
-
+    if (data.settings.background) data.settings.background_color = data.settings.background.color;
     if (data.custom) data.custom_defaults = data.custom;
+    // Have to explicitly check if undefined here as it is a boolean value
+    if (data.settings.custom_html_enabled !== undefined) {
+      data.settings.custom_enabled = data.settings.custom_html_enabled;
+    }
```

Comment:
If you have a global patching like that after fetching, you should have a global patching before sending changes. Otherwise use a widget-specific patching via `patchAfterFetch` and `patchBeforeSend`